src/widgets/options_widgets/dataset_options.py

```diff
@@ -0,0 +1,43 @@
+from typing import Dict
+
+from .options import Options
+from PyQt5.QtWidgets import QLabel, QLineEdit, QSpinBox
+
+
+class ClusteringBlobsDataOptions(Options):
+    def __init__(self):
+        super().__init__()
+        self.number_of_blobs_box = QSpinBox()
+        self.number_of_blobs_box.setRange(0, 20)
+        self.number_of_blobs_box.setValue(2)
+        self.layout.addRow(QLabel("Number of blobs:"), self.number_of_blobs_box)
+
+        self.sample_size_input = QLineEdit("50 30")
+        self.layout.addRow(QLabel("Sample sizes per blob:"), self.sample_size_input)
```

Comment:
You can add tooltip to inform what format of input is valid (you can use `QLabelWithTooltip` component from @gregori0o/improve-descriptions branch).

Reply:
great idea!